src/prg-c-d.asm

```diff
@@ -7,39 +7,29 @@
 ;   - The ending sequence with Mario sleeping and the cast roll
 ;
 
-MarioDream_Pointers:
-	.dw PPUBuffer_301
-	.dw MarioDream_Bed
-	.dw MarioDream_Bubble
-	.dw MarioDream_DoNothing
-	.dw MarioDream_EraseBubble1
-	.dw MarioDream_EraseBubble2
-	.dw MarioDream_EraseBubble3
-	.dw MarioDream_EraseBubble4
-	.dw MarioDream_EraseBubble5
-	.dw MarioDream_Palettes
+.include "src/mario_dream/dream_ptr_table.asm"
```

Comment:
maybe it's better to leave the pointers in this file and just separate out the ppu data?